mods/cnc/rules/infantry.yaml

```diff
@@ -15,6 +15,7 @@ E1:
 	Armament:
 		Weapon: M16
 	AttackFrontal:
+		Voice: Move
```

Comment:
It may be cleaner to define an Attack voice instead of overriding this everywhere.